### DISCUSSION TITLE
希望大佬做浏览器扩展版本

### DISCUSSION THREAD

**Discussion by anna1997411** [pot-app/pot-desktop#499] [General]
非常感谢大佬为爱发电做项目。希望未来有机会能做浏览器扩展版本

**Comment by MAHXVFX** [comment]
> 非常感谢大佬为爱发电做项目。希望未来有机会能做浏览器扩展版本

浏览器翻译你可以用沉浸式翻译呀，非常好用

**Comment by ajt2520** [comment]
建议开发浏览器插件，谢谢

**Reply by Pylogmon** [reply]
有这个必要吗？pot主打的就是全局取词，我为什么要把它再限制到浏览器里边。

**Comment by MC-dusk** [comment]
浏览器扩展一堆现成的不用，为什么还要在这提，无法理解

**Reply by MC-dusk** [reply]
防引流我就不提了，但是浏览器扩展做得好用的多得很

**Comment by z1027253689** [comment]
我用这个全局取词，就是想代替掉浏览器OCR不能全局取词的局限性，不是很理解为啥还要搞个浏览器版本
